.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  global:
+    CHANS_DEV: "-c pyviz/label/dev -c ioam -c bokeh -c conda-forge -c erdc -c defaults"
```

Comment:
That is quite a list of channels!  Do we really need all of them?

Reply:
I guess I'm expecting it to become something like `-c pyviz/label/dev -c erdc -c conda-forge` for developers, and `-c pyviz -c erdc -c conda-forge` for released packages. And then as we get more onto c-f and defaults, maybe releases will eventually not need extra channels (someone can just use defaults or c-f).